assets/javascripts/discourse/components/query-row-content.js

```diff
@@ -46,23 +57,15 @@ export default class QueryRowContent extends Component {
       }
 
       try {
-        return htmlSafe(
-          (t.template || this.args.fallbackTemplate)(ctx, params)
-        );
+        return htmlSafe((t.template || this.fallbackTemplate)(ctx, params));
```

Comment:
Probably something to do in a separate PR... but it would be great if we can get rid of the raw-hbs-based rendering here 🤔